operator/src/test/java/oracle/kubernetes/job/CreateDomainTest.java

```diff
@@ -1,67 +0,0 @@
-// Copyright (c) 2018, 2021, Oracle and/or its affiliates.
```

Comment:
I can't tell if this was an intentional deletion of a test or another accidental revert.

Reply:
This was intentional. It's been @Ignored for more than two years, so I figured we didn't really need it.